Software/vortex_ws/src/README.md

```diff
@@ -1 +1,28 @@
-All the ROS packages should be created in this fo;der
+## prerequisites¶
+#### Dependencies:
+- [Robot Operating System (ROS2)](https://index.ros.org/doc/ros2/Installation/Eloquent/Linux-Install-Debians/) (Install Eloquent as Debian Package).
+- [Colcon](https://index.ros.org/doc/ros2/Tutorials/Colcon-Tutorial/)
+#### You can  install Dependencies at the first time as below:
+
+* #### **System requirements¶**
+   **Target platforms for Eloquent Elusor are [(see REP 2000)](https://www.ros.org/reps/rep-2000.html):**
+    - **Tier 1: Ubuntu Linux - Bionic Beaver (18.04) 64-bit**
+    - **System setup**
+    - **Set locale**
```

Comment:
CAN not understand what is the meaning of this, can you please provide more details what is the use of this